components/Chips/src/ColorThemer/MDCChipViewColorThemer.h

```diff
@@ -0,0 +1,98 @@
+
+ @param colorScheme The color scheme to apply to @c MDCChipView.
+ @param inputChipView @c MDCChipView to which the color scheme should be applied.
+ */
++ (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
+                 toInputChipView:(nonnull MDCChipView *)inputChipView;
+
+/**
+ Applies a color scheme's properties to @c MDCChipView.
+
+ @param colorScheme The color scheme to apply to @c MDCChipView.
+ @param strokedInputChipView @c MDCChipView to which the color scheme should be applied.
+ */
++ (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
+          toStrokedInputChipView:(nonnull MDCChipView *)strokedInputChipView;
+
+/**
+ Applies a color scheme's properties to @c MDCChipView.
+
+ @param colorScheme The color scheme to apply to @c MDCChipView.
+ @param choiceChipView @c MDCChipView to which the color scheme should be applied.
+ */
++ (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
+                toChoiceChipView:(nonnull MDCChipView *)choiceChipView;
+
+/**
+ Applies a color scheme's properties to @c MDCChipView.
+
+ @param colorScheme The color scheme to apply to @c MDCChipView.
+ @param strokedChoiceChipView @c MDCChipView to which the color scheme should be applied.
+ */
++ (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
+         toStrokedChoiceChipView:(nonnull MDCChipView *)strokedChoiceChipView;
+
+/**
+ Applies a color scheme's properties to @c MDCChipView.
+
+ @param colorScheme The color scheme to apply to @c MDCChipView.
+ @param actionChipView @c MDCChipView to which the color scheme should be applied.
+ */
++ (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
+                toActionChipView:(nonnull MDCChipView *)actionChipView;
+
+/**
+ Applies a color scheme's properties to @c MDCChipView.
+
+ @param colorScheme The color scheme to apply to @c MDCChipView.
+ @param strokedActionChipView @c MDCChipView to which the color scheme should be applied.
+ */
++ (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
+         toStrokedActionChipView:(nonnull MDCChipView *)strokedActionChipView;
+
+/**
+ Applies a color scheme's properties to @c MDCChipView.
+
+ @param colorScheme The color scheme to apply to @c MDCChipView.
+ @param filterChipView @c MDCChipView to which the color scheme should be applied.
+ */
++ (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
+                toFilterChipView:(nonnull MDCChipView *)filterChipView;
+
+/**
+ Applies a color scheme's properties to @c MDCChipView.
+
+ @param colorScheme The color scheme to apply to @c MDCChipView.
+ @param strokedFilterChipView @c MDCChipView to which the color scheme should be applied.
+ */
++ (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
+         toStrokedFilterChipView:(nonnull MDCChipView *)strokedFilterChipView;
```

Comment:
Reading through the implementation I think we may be ok to just provide a single themer API for chips if all of the theme mappings are identical. If the mappings change in the future then we can/should break the API apart, but for now it's not providing any value.

Reply:
yeah it's only different between stroked and non stroked.

Reply:
Let's keep the API simpler then and just theme based on stroked/not stroked.